api/v1alpha1/hostedcluster_types.go

```diff
@@ -582,6 +582,12 @@ type PowerVSPlatformSpec struct {
 	// +immutable
 	AccountID string `json:"accountID"`
 
+	// CISInstanceCRN is the IBMCloud CIS Service Instance's Cloud Resource Name
+	// This field is immutable. Once set, It can't be changed.
+	//
+	// +immutable
+	CISInstanceCRN string `json:"cisInstanceCRN"`
```

Comment:
can we do any allowed input validation here? can we add an example to the // doc?

Reply:
Added a validation here. But this will be generated by the Infra creation process by getting the UUID of the base-domain passed. We already talked about CIS base domain in prerequisite section of PowerVS how-to guide.